tests/Integration/Behaviour/Features/Context/Domain/AliasFeatureContext.php

```diff
@@ -100,6 +104,39 @@ public function assertLastErrorIsInvalidAliasConstraint(): void
         $this->assertLastErrorIs(InvalidArgumentException::class);
     }
 
+    /**
+     * @When I delete alias :reference
+     *
+     * @param string $reference
+     */
+    public function deleteAlias(string $reference): void
+    {
+        /** @var string[] $aliasId */
+        $aliasId = $this->getSharedStorage()->get($reference);
+
+        try {
+            $this->getCommandBus()->handle(new DeleteAliasCommand(new AliasId((int) $aliasId)));
+        } catch (AliasException $e) {
+            $this->setLastException($e);
+        }
+    }
+
+    /**
+     * @When I bulk delete aliases :aliasReferences
+     *
+     * @param TableNode $aliasesList
+     */
+    public function bulkDeleteAlias(string $aliasReferences): void
+    {
+        try {
+            $this->getCommandBus()->handle(new BulkDeleteAliasCommand(
+                array_map('intval', $this->referencesToIds($aliasReferences))
```

Comment:
```suggestion
                $this->referencesToIds($aliasReferences)
```
shouldn't need to cast, all the ids should already be saved to storage as int.

Reply:
Somehow it comes as string id's

Reply:
"somehow" :D it depends what you save them when you do `$this->getShaedStorage()->set($reference, $id)`